evaluation/data/smops/C1152/test/view_C96.ncl

```diff
@@ -0,0 +1,51 @@
+begin
+
+  res                       = True              ; Plot mods desired.
+  res@gsnMaximize           = True              ; Maximize plot
+  res@gsnAddCyclic          = False             ; don't add a longitude cyclic point
+  res@cnFillOn              = True              ; color plot desired
+  res@cnFillOpacityF        = 0.6               ; apply a little opacity so we can see mesh lines better
+  res@cnFillMode            = "CellFill"
+  res@cnFillPalette         = "BlWhRe"
+  res@cnLinesOn             = False             ; turn off contour lines
+  res@cnLineLabelsOn        = False             ; turn off contour labels
+  res@tiMainFontHeightF     = 0.017
+  res@pmTitleZone           = 4                 ; move title down
+  res@mpFillOn              = False             ; turn off map fill
+  res@pmTickMarkDisplayMode = "Always"          ; nicer map tickmark labels
+  res@mpDataBaseVersion     = "LowRes"       ; better map outlines
+  res@mpOutlineBoundarySets = "Geophysical"
+  res@mpOutlineOn           = True
+  res@mpNationalLineThicknessF    = 2.0      ; for better looking images
+  res@mpGeophysicalLineThicknessF = 2.0
+  res@mpCountyLineThicknessF      = 2.0
+  res@mpProvincialLineThicknessF  = 2.0
+  res@gsnRightString        = ""
+  res@gsnLeftString         = ""
+  res@lbLabelBarOn          = True             ; Will turn on in panel
+
+  if(False) then
+    wks = gsn_open_wks("x11","blah") 
+  else
+    wks = gsn_open_wks("png","blah") 
+  end if
+
+  infile = addfile("/scratch2/NCEPDEV/land/Michael.Barlage/forcing/C96/vector/ufs-land_C96_hr3_corners.nc","r")
```

Comment:
does this file exist in the data directory? if so, we should use that one instead of the one in my directory. if it doesn't exist, then we should put a copy there

Reply:
It does not exist in /scratch2/NCEPDEV/land/data/forcing/era5/static/C96. I do not have write permission for that directory. Could you please copy /scratch2/NCEPDEV/land/Michael.Barlage/forcing/C96/vector/ufs-land_C96_hr3_corners.nc to that directory?  Thank you.

Reply:
I put the files in the domains directory like this:
/scratch2/NCEPDEV/land/data/evaluation/domains/C96/fix_20231027/ufs-land_C96_hr3_corners.nc

Reply:
Use this file to modify ncl scripts. I tested them and they work.